apps/ais/src/app/solid-core-config.ts

```diff
@@ -11,4 +12,5 @@ export const coreConfig: SolidCoreConfig = {
   profileTour: guidedTour.profile,
   glossaryLogo: 'assets/svg/glossary.svg',
   feedbackLogo: 'assets/svg/feedback.svg',
+  landingContent: landingContent,
```

Comment:
besser in landingBannerContent umbenennen (konsistent mit der app-spezifischen component)